plugins/intsights/icon_intsights/actions/takedown_request/action.py

```diff
@@ -0,0 +1,19 @@
+import insightconnect_plugin_runtime
+from .schema import TakedownRequestInput, TakedownRequestOutput, Input, Output, Component
+
+# Custom imports below
+from insightconnect_plugin_runtime.helper import clean
+
+
+class TakedownRequest(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="takedown_request",
+            description=Component.DESCRIPTION,
+            input=TakedownRequestInput(),
+            output=TakedownRequestOutput(),
+        )
+
+    def run(self, params={}):
+        self.connection.client.takedown_request(params.get(Input.ALERT_ID), params.get(Input.TARGET))
+        return clean({Output.STATUS: True})
```

Comment:
```suggestion
        return {Output.STATUS: True}
```
No need for clean here if this is always True.